plugins/generator-1.15.2/forge-1.15.2/templates/json/txblock/end_rod.json.ftl

```diff
@@ -0,0 +1,49 @@
+<#-- @formatter:off -->
+{
+    "parent": "block/block",
+    "display": {
+        "head": {
+            "rotation": [ -60, 0, 0 ],
+            "translation": [ 0, 5, -9],
+            "scale":[ 1, 1, 1]
+        },
+        "thirdperson_righthand": {
+            "rotation": [ 0, 0, 0 ],
+            "translation": [ 0, 0, 0],
+            "scale": [ 0.375, 0.375, 0.375 ]
+        }
+    },
+    "ambientocclusion": false,
+    "textures": {
+        "end_rod": "${modid}:blocks/${data.texture}",
+        <#if data.particleTexture?has_content>"particle": "${modid}:blocks/${data.particleTexture}"
```

Comment:
Change this to a shorter ternary operator version as you can see for example here: https://github.com/MCreator/MCreator/blob/a80207b22c8c342c85e4d3e4e9f9564a60bfcdfc/plugins/generator-1.16.5/forge-1.16.5/templates/json/block.json.ftl#L5

Same for 1.16

Reply:
Done.